desktop/ui/src/main/java/org/datacleaner/windows/ResultWindow.java

```diff
@@ -359,15 +398,18 @@ protected JComponent getWindowContent() {
 
         final DCBannerPanel banner = new DCBannerPanel(imageManager.getImage("images/window/banner-results.png"),
                 bannerTitle);
-        _tabbedPane.bindTabTitleToBanner(banner);
+        _tabbedPane.addListener(new VerticalTabbedPane.Listener() {
+            @Override
+            public void stateChanged(int newIndex, Tab<?> newTab) {
+                banner.setTitle2(newTab.getTitle());
+                banner.updateUI();
+            }
+        });
 
         for (JComponent pluggableButton : _pluggableButtons) {
             banner.add(pluggableButton);
```

Comment:
Yes those are also good improvements, but I am only trying to make it simple for you to get this PR merged :-)

Currently the only pluggable button is this one:

![image](https://cloud.githubusercontent.com/assets/291450/8649126/626433de-2967-11e5-805a-f9e0d44b1264.png)